dependency-alerts/frontend-repos.json

```diff
@@ -1,8 +1,6 @@
 [
-  "Sage-Bionetworks/portals",
-  "Sage-Bionetworks/Synapse-React-Client",
+  "Sage-Bionetworks/synapse-web-monorepo",
   "Sage-Bionetworks/synapse-oauth-signin",
-  "Sage-Bionetworks/markdown-it-synapse-server",
```

Comment:
We do not maintain this project, but the backend may or may not use this for markdown-it processing (used to use for discussion forum email notifications?)